docs/src/pages/components/Map/WebMap.mdx

```diff
@@ -15,23 +15,23 @@ import Readme from 'backpack/packages/bpk-component-map/README.md';
 ## Default
 
 <PresentationBlock>
-  <SimpleExample />
+  <iframe width="800" height="600" src="https://backpack.github.io/storybook/iframe.html?id=bpk-component-map--simple"></iframe>
```

Comment:
Just waiting for the PR build to finish deploying as want to check the size this will render out on the docs site

Reply:
So that it doesn't break out of its container when on mobile can you change the width to be:

```suggestion
  <iframe style={{width: '100%'}} height="600" src="https://backpack.github.io/storybook/iframe.html?id=bpk-component-map--simple"></iframe>
```